### PR TITLE
Use python 3.6 on bionic.

### PR DESCRIPTION
**Pull request by nuclearsandwich**
I'm not sure what the optimal way to get the desired python interpreter version here is.

For now let's continue to hard code it.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ project(ros_workspace)
 
 find_package(ament_cmake_core REQUIRED)
 
-set(PYTHON_INSTALL_DIR "lib/python3.5/site-packages")
+set(PYTHON_INSTALL_DIR "lib/python3.6/site-packages")
```
Comment (mikaelarguedas) [inline CMakeLists.txt:6]: We should be able to get this from python directly. e.g. (not saying it's the best way..) getting the version by calling `execute_process` with something like:
`${PYTHON_EXECUTABLE} -c from distutils import sysconfig; sysconfig.get_python_version()` 

On my xenial machine: 
```
$ python3 -c "from distutils import sysconfig; print(sysconfig.get_python_version())"
3.5
```

**Comment by mikaelarguedas** [comment]
Question: Should this target `master` instead of `latest` ?
Not sure how the branching works on this repo

**Review by mikaelarguedas — APPROVED** [review]
lgtm, dropped a suggestion on how to make this less hard-coded

**Comment by nuclearsandwich** [comment]
> Question: Should this target master instead of latest ?
> Not sure how the branching works on this repo

`master` branch is vestigial and has just been deleted to avoid confusion. `latest` is the correct "trunk" branch.

File: CMakeLists.txt (hunk 1 above)
Comment (nuclearsandwich) [reply CMakeLists.txt:6]: This looks good. I'll give this a try.

**Comment by nuclearsandwich** [comment]
Closing in favor of #4.

**Pull request closed without merging**